components/admission-webhook/manifests/base/crd.yaml

```diff
@@ -8,46 +8,62 @@ spec:
     kind: PodDefault
     plural: poddefaults
     singular: poddefault
+  preserveUnknownFields: false
   scope: Namespaced
-  version: v1alpha1
-  validation:
-    openAPIV3Schema:
-      properties:
-        apiVersion:
-          type: string
-        kind:
-          type: string
-        metadata:
-          type: object
-        spec:
+  versions: 
+    - name: v1alpha1
```

Comment:
Maybe a dumb question but why is this one `v1alpha1`?

Reply:
cause the version of the CRD (poddefault) did not change, we only changed the k8s CRD version.